spec/classes/init_spec.rb

```diff
@@ -20,7 +20,9 @@
                        when 'Debian'
                          'tftpd-hpa'
                        when 'FreeBSD'
-                           'tftp-hpa'
+                         'tftp-hpa'
+                       when 'Archlinux'
+                         'tftp-hpa'
```

Comment:
I know I did this, but I think now that should be just tftp-hpa for the general else case.